.azure-pipelines/azure-pipelines-repd-build-variables.yml

```diff
@@ -0,0 +1,5 @@
+variables:
+  ${{ if and(ge(variables['Build.SourceBranchName'], '202012'), le(variables['Build.SourceBranchName'], '299999')) }}:
+    VERSION_CONTROL_OPTIONS: 'SONIC_VERSION_CONTROL_COMPONENTS=deb,py2,py3,web,git,docker'
+  ${{ if or(lt(variables['Build.SourceBranchName'], '202012'), gt(variables['Build.SourceBranchName'], '299999')) }}:
```

Comment:
What happen if the SourceBranchName is not a number? Should we explicitly handle this case?

Reply:
@qiluo-msft , we use string comparing, will skip the reproducible build feature.

Reply:
`VERSION_CONTROL_OPTIONS` is not defined in that case. Is it going to be a problem?

Reply:
It is depended how to use the variable.
```
Micro: $(VERSION_CONTROL_OPTIONS)   --- not work
Template: ${{ variables.VERSION_CONTROL_OPTIONS }}  --- empty
Runtime: $[variables.VERSION_CONTROL_OPTIONS ]  --- empty
```
Currently, we are good, we use template way.